### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by andrewsilke**
fixing a few bugs with layouts in zoocore layouts.py and a couple of other small tweaks.

### CODE REVIEW

File: zoo/libs/pyqt/widgets/layouts.py [hunk 1]
```diff
@@ -364,7 +368,10 @@ def _onTextChanged(self):
         """updates the text, should also update the dict
         """
         # todo: check that the methods below work, not tested
-        self.textChanged.emit(tuple([float(i.value()) for i in self._widgets.text()]))
+        valueList = list()
+        for axis in self._widgets:
+            valueList.append(self._widgets[axis].text())
```
Comment (dsparrow27) [inline zoo/libs/pyqt/widgets/layouts.py]: `valueList = [self._widgets[axis].text() for axis in self._widgets]`

File: zoo/libs/pyqt/widgets/layouts.py [hunk 2]
```diff
@@ -46,16 +46,20 @@ def LineEdit(text="", placeholder="", parent=None, toolTip="", editWidth=None, i
     # todo: STYLESHEET hardcoded color & margins here as a temp workaround, this should be in stylesheets
     textBox.setStyleSheet("QLineEdit {background: rgb(27, 27, 27);}")
     if inputMode == "float":  # todo: this should be a constant
```
Comment (dsparrow27) [inline zoo/libs/pyqt/widgets/layouts.py]: Can you add some inline comments from this function?

File: zoo/libs/pyqt/widgets/layouts.py [hunk 3]
```diff
@@ -382,14 +389,18 @@ def value(self):
         :return textValues: a tuple of string values from each QLineEdit textbox
         :rtype textValues: tuple(str)
         """
-        return tuple([float(i.value()) for i in self._widgets.text()])
+        valueList = list()
+        for axis in self._widgets:
+            valueList.append(self._widgets[axis].text())
```
Comment (dsparrow27) [inline zoo/libs/pyqt/widgets/layouts.py]: `valueList = [self._widgets[axis].text() for axis in self._widgets]`

File: zoo/libs/pyqt/widgets/layouts.py (hunk 1 above)
Comment (andrewsilke) [reply zoo/libs/pyqt/widgets/layouts.py]: Yeah still finding one-liners difficult. ta

File: zoo/libs/pyqt/widgets/layouts.py (hunk 3 above)
Comment (andrewsilke) [reply zoo/libs/pyqt/widgets/layouts.py]: fixed

File: zoo/libs/pyqt/widgets/layouts.py (hunk 1 above)
Comment (andrewsilke) [reply zoo/libs/pyqt/widgets/layouts.py]: done

**Pull request merged**